src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -1079,7 +1079,7 @@ void ShenandoahVerifier::verify_after_concmark() {
           _verify_cset_none,           // no references to cset anymore
           _verify_liveness_complete,   // liveness data must be complete here
           _verify_regions_disable,     // trash regions not yet recycled
-          _verify_size_exact,          // expect generation and heap sizes to match exactly
+          sizeness,          // expect generation and heap sizes to match exactly
```

Comment:
So... the comment is outdated? I suggest to split `ShenandoahVerifier::verify_after_concmark` into two methods, and clearly point out the expected state in comments then.

Reply:
Done.